frontend/src/services/friendships/service.ts

```diff
@@ -0,0 +1,22 @@
+import { client } from "../supabase/client";
+import { UpdateFriendShipRequest } from "./type";
+
+class _friendService {
+  updateFriendshipIdByFriendemail = async ({
+    friendId,
+    friendEmail,
+  }: UpdateFriendShipRequest) => {
+    const { error } = await client
+      .from("Friendships")
+      .update({ friendId })
+      .eq("friendEmail", friendEmail);
```

Comment:
I'm sorry. Because the current code is joined with domain logic well and looks difficult to write the logic in useSignup. So I decided on your suggestion. Thanks.